data/taipei/middle-name-cafe.md

```diff
@@ -0,0 +1,47 @@
+---
+updated: 2021-01-26
```

Comment:
If you're adding a new place for the first time, use `added: <date>` instead of `updated: <date>`